__tests__/index.test.ts

```diff
@@ -2165,6 +2165,257 @@ describe('Auth0', () => {
     });
   });
 
+  describe('buildAuthorizeUrl', () => {
```

Comment:
Why are there tests affected? I think we want to ensure we are only touching new tests or updating tests that needs to be updated.

Reply:
I'm not sure why these are being marked as changed.. whitespace changes? I'll try to resolve.

Reply:
I think this happened in this commit: 8d2df480d6a245003e71b0584dc010baba294a17

I think it's just the result of how git aligned the changes when I erroneously messed up the test blocks. After fixing, it's still not lining up correctly. 

Not sure how to fix so the PR doesn't suggest the tests were changed? Any suggestions?

Reply:
This is not just a whitespace issue, the entire describe block for `buildAuthorizeUrl ` exists twice.

Reply:
Thanks - fixed in 72253f6